src/fidocadj/dialogs/OriginCircuitPanel.java

```diff
@@ -46,8 +46,8 @@ public final class OriginCircuitPanel extends CircuitPanel
 
     // x and y coordinates of the origin in logical units.
     // TODO: improve data encapsulation (these should be private).
```

Comment:
You can remove the TODO comment (if you haven't done it yet in the next commits I haven't yet looked at).